### PR TITLE
Javascript3 week1

### PR DESCRIPTION
**Pull request by natata08**
I implemented the following features:
- Fetching the initial questions data from API.
- Adding the ability to sort the questions alphabetically.
- Adding the ability to sort the questions randomly.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -445,3 +347,30 @@ function resetGame() {
...
+  });
+};
+
+const sortRandomly = (questions) =>
+  [...questions].sort(() => Math.random() - 0.5);
+
+function sortQuestions() {
+  const sortType = document.getElementById('sort-select').value;
+  const sortedQuestions =
+    sortType === 'alphabetical'
```
Comment (babak-f) [inline script.js:372]: Good!

File: styles.css [hunk 2]
```diff
@@ -212,6 +212,26 @@ form {
...
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  gap: 1.5rem;
+  flex-wrap: wrap;
+}
+
+#sort-select {
+  padding: 0.625rem 0.7rem;
```
Comment (babak-f) [inline styles.css:230]: Usually it's a good idea to keep spacings a multiple of 0.25rem, to make the overall layout more consistent

Comment (babak-f) [reply styles.css:230]: But otherwise, good job for using rem 👍

**Review by babak-f — APPROVED** [review]

File: script.js (hunk 1 above)
Comment (natata08) [reply script.js:372]: Thank you for the review!

**Pull request merged**